app/controllers/admin/custom_files_controller.rb

```diff
@@ -0,0 +1,65 @@
+module Admin
+  class CustomFilesController < Admin::BaseController
```

Comment:
__[Reek]__

[InstanceVariableAssumption] Admin::CustomFilesController assumes too much for instance variable '@custom_file'


<!-- Comment by [SideCI](https://sideci.com) -->